pkg/bwagreement/server.go

```diff
@@ -51,14 +56,16 @@ type DB interface {
 // Server is an implementation of the pb.BandwidthServer interface
 type Server struct {
 	db     DB
```

Comment:
I think renaming `db` to `bwdb` might make it more clear since there are two databases on this server